tools/importers/helpers/modifiers.ts

```diff
@@ -6,6 +6,28 @@ export const pascalCase = (str: string) => {
 		.replace(/\s+/g, '');
 };
 
+export const camelize = (str: string) => {
+	return str
+		.replace(/(?:^\w|[A-Z]|\b\w)/g, function (word, index) {
+			return index === 0 ? word.toLowerCase() : word.toUpperCase();
+		})
+		.replace(/\s+/g, '');
+};
+
+export const kebabize = (str) => {
+	if (str.length < 4) {
```

Comment:
I don't like the fact that this method depends on hardcoded string length. What if someone decides to add `10xl` in token name?

Maybe we should call this method in different moment in the process, when we did or didn't perform some operations on strings?

Reply:
I changed this condition to more precise.